Revamped patient details page UI

patelaryan7751 opened the pull request:
Fixes #2851 

# Bugs Fixed 

## Revamped patient details page UI and changed page name to the patient details page

![image](https://user-images.githubusercontent.com/59426397/175799136-d1dac253-55fb-4c3f-9d44-690f2c0650d7.png)

cp-Coder reviewed (approved):
LGTM

sainAk commented on src/Components/Patient/PatientHome.tsx:
```diff
@@ -1088,7 +1089,184 @@ export const PatientHome = (props: any) => {
+                      disabled={
+                        !patientData.is_active ||
+                        !(
+                          patientData?.last_consultation?.facility == facilityId
```
```suggestion
                          patientData?.last_consultation?.facility === facilityId
```
https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=9682972

nihal467 commented:
@patelaryan7751 test failed

![3](https://user-images.githubusercontent.com/57055998/175970571-872ec3c2-6b9a-4f9b-850f-88f2aaece1b0.jpg)

It is not patient details , it is sample test history

patelaryan7751 commented:
@nihal467  I have changed the title to Sample Test History
![image](https://user-images.githubusercontent.com/59426397/175997058-bb924861-31ab-4e9c-adb3-97c96061fdc8.png)
I also changed the title "Covid Suspect Detail" to "Patient Detail"
![image](https://user-images.githubusercontent.com/59426397/175997241-0a71d95d-e0db-4203-8de7-386bafcad466.png)

nihal467 commented:
@patelaryan7751 

![7](https://user-images.githubusercontent.com/57055998/175998631-f7a39091-22ca-4ba6-b50a-6993f2beb78f.jpg)

- [ ] add a white box, in that mention under-sample test history that there is no data available error message

![7a](https://user-images.githubusercontent.com/57055998/175998644-b802bcff-06b1-434f-a54a-28b9fd9bd9e0.jpg)

- [ ] align the badges to the top contents

patelaryan7751 commented:
Hey @nihal467 I have made the above said changes
### Added a white box for both sample details and consultation details
![image](https://user-images.githubusercontent.com/59426397/177477135-39e6cb82-fcc4-4786-8c02-798682c2aa65.png)

### Aligned all badges in patient card
![Screenshot from 2022-07-06 11-06-31](https://user-images.githubusercontent.com/59426397/177477327-974344ea-eb91-40ed-bf6a-bbe1e8a4855e.png)

nihal467 commented:
@patelaryan7751 
![30](https://user-images.githubusercontent.com/57055998/177490425-a692d20b-8242-46d0-85df-e488a6c0f433.jpg)

Add District too, because in Manipur, the local body doesn't mention the district name, as we have in Kerala

patelaryan7751 commented:
Hey, @nihal467 added district in location card
![image](https://user-images.githubusercontent.com/59426397/177990115-5a76efb7-bb9d-497a-aace-dc65d0c0d0f8.png)

nihal467 commented:
test approved
